tests/security_test.go

```diff
@@ -0,0 +1,143 @@
+package tests
+
+import (
+	"github.com/mesosphere/kudo-spark-operator/tests/utils"
+	log "github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"testing"
+)
+
+func TestDefaultServiceAccounts(t *testing.T) {
```

Comment:
Looks like these three tests methods(TestOperatorServiceAccount, TestSparkServiceAccount and this one) contains same logic. mb it’s make sense to extract a template method and invoke it with appropriate parameters in the tests methods, WDYT?

Reply:
Also, perhaps, it makes sense consider the single test method instead of three separate, it could looks like
```suggestion
func TestServiceAccounts(t *testing.T) {
        t.Run("TestDefaultServiceAccounts", testServiceAccount(namespace1, driver1, operator1, t));
        t.Run("TestOperatorServiceAccount", testServiceAccount(namespace2, driver2, operator2, t));
        t.Run("TestSparkServiceAccount", testServiceAccount(namespace3, driver3, operator3, t));
}

func testServiceAccount(namespace string, driverSA string, operatorSA string, t *testing.T) {
     
    return func(t *testing.T) {
 // the test method template
 }
}
```

Reply:
Makes sense. I refactored the cases into sub-tests.